src/mono/wasm/runtime/logging.ts

```diff
@@ -62,8 +62,8 @@ export function mono_wasm_symbolicate_string(message: string): string {
 
 export function mono_wasm_stringify_as_error_with_stack(err: Error | string): string {
     let errObj: any = err;
-    if (!(errObj instanceof Error)) {
-        errObj = new Error(errObj);
+    if (!errObj || !errObj.stack || !(errObj instanceof Error)) {
+        errObj = new Error(errObj || "Unknown error");
```

Comment:
It's not a big deal but we're checking twice if `errObj` is empty, in if and when passing arg to Error. Whatever you feel more comfortable with.
```suggestion
    if (!errObj || !errObj.stack) {
        errObj = new Error("Unknown error");
    } else if (!(errObj instanceof Error)) {
        errObj = new Error(errObj");
    }
```

Reply:
I prefer the original dense version :)